static/index.html

```diff
@@ -52,17 +52,15 @@
       </div>
     </div>
     <div class="heading">
-      <img src="/static/images/globe.svg" class="logo" alt="World driven logo" height="60" width="60">
-      <h1>World Driven</h1>
+      <img src="/static/images/logo_world_driven.svg" class="logo" alt="World driven logo">
```

Comment:
It would be nice if the white background of this svg was removed. Not strictly part of the issue this PR resolves, but it might be worth doing while were updating it anyway.